api/control/v1alpha1/types.go

```diff
@@ -257,3 +257,46 @@ type BindingList struct {
 	metav1.ListMeta `json:"metadata,omitempty"`
 	Items           []Binding `json:"items"`
 }
+
+// CustomClean describes how to clean a resource.
+// That is, how to clean objects of a given resource.
+// The name of this object is the `resource.apiGroup` of the resource
+// that it is about; for example, `appwrappers.workload.codeflare.dev`.
+// The cleaning specified here is in addition to whatever cleaning
+// is built into KubeStellar for that resource.
+//
+// +genclient
+// +genclient:nonNamespaced
+// +kubebuilder:object:root=true
+// +kubebuilder:resource:scope=Cluster,shortName={cc},categories={all}
+type CustomClean struct {
```

Comment:
Why only Cleaning?    Why can't we add or change fields ?

Reply:
you should be seeing that the latest revision uses a different name.

Reply:
Thanks